Homework-Feb-29/BWT/SuffixArray.cs

```diff
@@ -0,0 +1,44 @@
+namespace BWT;
+
+static class SuffixArray
+{
+    public static List<int> BuildSuffixArray(string sourceString)
+    {
+        var suffixArray = new List<int>();
+        for (var i = 0; i < sourceString.Length; ++i)
+        {
+            suffixArray.Add(i);
+        }
+        SortSuffixArray(sourceString, suffixArray);
+        return suffixArray;
+    }
+
+    private static int CompareSuffix(string sourceString, 
+        int suffixStartPos1, int suffixStartPos2)
+    {
+        for (int i = 0; i < sourceString.Length; ++i)
```

Comment:
Тут тоже можно var писать

```suggestion
        for (var i = 0; i < sourceString.Length; ++i)
```